application/Profile/DBProfile.py

```
@@ -118,27 +146,37 @@ class Session(db.Model):
 
             @validates('nickname')
             def validate_nickname(self, key, nickname):
-                assert len(nickname) <= 8, \
-                    'Entered nickname is too long'
+                nickname_too_long = 'Entered nickname is too long'
+                try:
+                    assert len(nickname) <= 8
```

Ruff

Use of `assert` detected

(S101)